src/components/views/rooms/MessageComposer.js

```diff
@@ -285,6 +285,10 @@ export default class MessageComposer extends React.Component {
             action: 'view_room',
             highlighted: true,
             room_id: replacementRoomId,
+
+            // Try to join via the server that sent the event. This converts $something:example.org
+            // into a server domain by splitting on colons and ignoring the first entry ("$something").
+            via_servers: [this.state.tombstone.getId().split(':').splice(1).join(':')],
```

Comment:
Hmm, let's see if I understand this... `this.state.tombstone.getId()` would be `$something:example.org`, and then `.split(':')` is `[ "$something", "example.org" ]`. What about the following suggestion then?

```suggestion
            via_servers: [this.state.tombstone.getId().split(':')[1],
```

Reply:
Server names can be IPv6 addresses or contain ports, so `[1]` would be incorrect.

```
> '$something:example.org:8080'.split(':');
[ '$something', 'example.org', '8080' ]
> '$something:example.org:8080'.split(':').splice(1).join(':');
'example.org:8080'
```